src/pages/Home.test.js

```diff
@@ -0,0 +1,9 @@
+import { render } from "@testing-library/react"
+import Home from "./Home"
+
+describe("<Home />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(<Home />, div)
+  })
```

Comment:
This isn't a test just a set up for some test. Perform an action or better yet, an assertion.